src/Runner.Worker/StepsRunner.cs

```diff
@@ -76,6 +75,13 @@ public async Task RunAsync(IExecutionContext jobContext)
                 // Start
                 step.ExecutionContext.Start();
 
+                // Expression functions
```

Comment:
look into moving this to execution context

Reply:
@TingluoHuang executionContext.CreateChild is too early, otherwise would be available during JobExtension.InitializeJob. Let me know what you think. I think here in steps runner makes sense, since these functions should be available until steps start running.